core/src/executor/select/mod.rs

```diff
@@ -124,8 +125,18 @@ pub async fn select_with_labels<'a>(
         order_by,
     } = match &query.body {
         SetExpr::Select(statement) => statement.as_ref(),
-        _ => {
-            return Err(SelectError::Unreachable.into());
+        SetExpr::Values(Values(values_list)) => {
+            let limit = Limit::new(query.limit.as_ref(), query.offset.as_ref())?;
+            let labels = values_list
+                .iter()
+                .enumerate()
+                .map(|(i, _)| format!("column{}", i + 1))
+                .collect::<Vec<_>>();
+            let rows = Row::to_rows(values_list)?;
```

Comment:
Instead of adding `Row::to_rows`, how about including
![image](https://user-images.githubusercontent.com/2025065/179401022-524b5822-5594-47d0-8fbb-f05a79ee9870.png)
TryFrom impl in here?

If we have more than a single case which requires conversion from `Vec<Expr>` to `Row`, then we can consider to make this as a shared function. But currently it is only for evaluating `SetExpr::Values` in `select` module, so it would be  enough to be located only for here.